content/blog/2016/2016-03-02-gsoc2016-announcement.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: "Jenkins participates in Google Summer of Code 2016"
+tags:
+- general
+- jenkins2
+- gsoc
+author: oleg_nenashev
```

Comment:
Will you make sure to add an author document to [content/_data/authors](https://github.com/jenkins-infra/jenkins.io/tree/master/content/_data/authors).

This isn't currently being used but will in the near future :)